io/eolearn/io/sentinelhub_service.py

```diff
@@ -158,12 +156,7 @@ def _add_meta_info(self, eopatch, request_params, service_type):
 
         for param, eoparam in zip(['time', 'time_difference', 'maxcc'], ['time_interval', 'time_difference', 'maxcc']):
             if eoparam not in eopatch.meta_info:
-                if param == 'time':
-                    value = request_params[param]
-                    eopatch.meta_info[eoparam] = value if isinstance(value, str) else [
-                        iso_to_datetime(parse_time(x)) if isinstance(x, str) else x for x in value]
-                else:
-                    eopatch.meta_info[eoparam] = request_params[param]
+                eopatch.meta_info[eoparam] = request_params[param]
```

Comment:
If I understand correctly, `request_params['time']` now always returns instances containing `datetime`?

Reply:
No, it can return either str or datetime, whatever the user provided as input.